src/utils/productFormUtils.ts

```
@@ -0,0 +1,103 @@
+  'GOOD': 'GOOD'
+};
+
+/**
+ * 폼 데이터를 API 요청 형식으로 변환
+ * @param formData - 폼에서 입력된 데이터
+ * @param images - 업로드된 이미지 배열
+ * @returns ProductRegistrationRequest
+ */
+export const transformFormDataToApiRequest = (
+  formData: Record<string, string>,
+  images: any[] = []
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)